rakelib/changelog.rake

```diff
@@ -37,7 +37,7 @@ class Changelog
     return if master_update?
 
     contents = File.read(changelog_file)
-    return unless contents =~ /^## master \(unreleased\)$/
+    return unless /^## master \(unreleased\)$/.match?(contents)
```

Comment:
Why is this needed?

Reply:
I'll revert it. Its change is related to rubocop warnings.